React/Views/RCTRefreshControl.m

```diff
@@ -31,6 +32,18 @@ - (instancetype)init
 - (void)layoutSubviews
 {
   [super layoutSubviews];
+  
+  // UIRefreshControl is managed so avoid changing it directly
+  // This method may break in future versions of iOS (subview structure may change)
+  // And self.transform won't shift title text
+  if(_isInitialRender) {
```

Comment:
Taking a look at this PR again. I'm concerned about the code here. It's been a while since I've worked on iOS and I am not sure about what current best practices are. @mmmulani do you approve of this method for finding the progress view subview?